brunch-config.js

```diff
@@ -52,6 +52,10 @@ exports.config = {
     babel: {
       // Do not use ES6 compiler in vendor code
       ignore: [/web\/static\/vendor/]
+    },
+    cleancss: {
+      keepSpecialComments: false,
+      removeEmpty: true
```

Comment:
@nburkley not sure if this is right, but tried to set up per readme at https://github.com/brunch/clean-css-brunch